dockers/docker-fpm-frr/bgpd.conf.j2

```diff
@@ -0,0 +1,64 @@
+!
+{% block banner %}
+! =========== Managed by sonic-cfggen DO NOT edit manually! ====================
+! generated by templates/quagga/bgpd.conf.j2 using minigraph_facts.py
+! file: bgpd.conf
+!
+{% endblock banner %}
+!
+{% block system_init %}
+hostname {{ inventory_hostname }}
+password zebra
+log syslog informational
+log facility local4
+! enable password {# {{ en_passwd }} TODO: param needed #}
+{% endblock system_init %}
+!
+{% block bgp_init %}
+!
+! bgp multiple-instance
```

Comment:
Currently config-CLIs for Quagga and FRR are not identical (to say the least) -- there's an increasing feature gap. Overtime, as more features are added to FRR, this gap will become larger, so I would definitely vote for having different 'j2' templates for both routing-stacks.